components/import-action/menu/menu.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import { Icon } from "@teambit/evangelist.elements.icon";
+import { Registry, Import, Install } from "../tab-content";
+import { Tab } from "../tab";
+import styles from "./menu.module.scss";
+
+export type MenuProps = {
+  /**
+   * package link to be copied
+   */
+  packageLink: string;
+  /**
+   * import link to be copied
+   */
+  bitLink: string;
+  /**
+   * registry link to be copied
+   */
+  registryName: string;
+  /**
+   * component name to be presented
+   */
+  componentName: string;
+  /**
+   * currently active tab
+   */
+  activeTab: string;
+  /**
+   * change currently active tab
+   */
+  setActiveTab: (active: string) => void;
+} & React.HTMLAttributes<HTMLDivElement>;
+
+export function Menu({
+  packageLink,
+  bitLink,
+  registryName,
+  componentName,
```

Comment:
consider passing the entire componentId, and getting relevant information from that, instead of passing down strings

Reply:
for this I will need the entire `componentModel` because the package name comes separately. its a bit excessive to pass all that, no?